src/main/java/com/querydsl/webhooks/GithubReviewWindow.java

```diff
@@ -60,65 +65,55 @@
  *
  * @author Shredder121
  */
+@EnableCaching
 @SpringBootApplication
 public class GithubReviewWindow {
 
     private static final Logger logger = LoggerFactory.getLogger(GithubReviewWindow.class);
 
-    private static final GitHub github;
-
     private final TaskScheduler taskScheduler = new ConcurrentTaskScheduler();
 
-    static {
-        try {
-            github = GitHub.connect();
-        } catch (IOException ex) {
-            throw Throwables.propagate(ex);
-        }
-    }
-
     public static void main(String... args) {
         GHEventApiServer.start(GithubReviewWindow.class, args);
     }
 
-    @Bean
-    public PullRequestHandler reviewWindowHandler(Environment environment) {
-        Duration defaultReviewWindow = Duration.parse(environment.getRequiredProperty("duration")); //duration is the default window
-        Map<String, ScheduledFuture<?>> asyncTasks = Maps.newConcurrentMap();
+    @Autowired
+    private GitHub gitHub;
```

Comment:
It's not _really_ necessary, but can be explored.
It already has its own well-known configuration (put it in the env, or in `~/.github`), so getting it up and running is already easy.
Getting it up and running easily in a Spring Boot context doesn't really seem like much added benefit in my opinion.